services/121-service/src/payments/payments.service.ts

```diff
@@ -547,6 +583,33 @@ export class PaymentsService {
     );
   }
 
+  // TODO: Needed to add _'s to allow unused parameters, as this function is not yet implemented
+  private async createAndAddIntersolveVisaTransferJobs(
+    referenceIds: string[],
+    _programId: number,
+    _paymentAmount: number,
+    _paymentNumber: number,
+  ): Promise<void> {
```

Comment:
RE: discussion via chat, this is essentially what I am suggesting we get into the habit of doing, instead of what is currently proposed:

```suggestion
  private async createAndAddIntersolveVisaTransferJobs({
    referenceIds,
    _programId,
    _paymentAmount,
    _paymentNumber,
  }: {
    referenceIds: string[],
    _programId: number,
    _paymentAmount: number,
    _paymentNumber: number,
  }): Promise<void> {
```

Reply:
Ok, interesting. I am having difficulty grasping what the effect of this is, just by looking at this code change. I do not really like the duplication of all parameter names, as in: does not look elegant :)
And then, does this have some benefit in VS Code when calling the function?
And what if the function would have only 1 or 2 parameters, then still use the options object? In short: always wrap all input parameters in an object?